src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/ReviewTransferNoteViewModelTests.cs

```diff
@@ -81,9 +81,11 @@ public void ReviewEvidenceNoteViewModel_ShouldBeInitialisedWithHintItems()
             //assert
             model.HintItems.ElementAt(0).Value.Should().Be(null);
             model.HintItems.ElementAt(0).Key.Should().Be("Approve evidence note transfer");
-            model.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note transfer if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note transfer");
+            //model.HintItems.ElementAt(1).Value.Should().Be("Reject an evidence note transfer if the evidence has been sent to you by mistake or if there is a large number of updates to make that it is quicker to create a new evidence note transfer");
```

Comment:
@GuidoWorkDev  Remove all of the commented out code

Reply:
run-out of adjectives